src/App.jsx

```diff
@@ -50,13 +53,44 @@ function App() {
       <div>
         <DoiBadge doi_id="ab-cd" />
       </div>
+      <DoiBadge
+        doi="custom-doi/10.ab.cd"
+        label="custom-label"
+        color="#a2e5b7"
+      />
+      <span>ToggleSwitches</span>
       <div>
-        <DoiBadge
-          doi="custom-doi/10.ab.cd"
-          label="custom-label"
-          color="#a2e5b7"
+        <ToggleSwitchBig switchLength="40px" />
+      </div>
+      <div>
+        <ToggleSwitchBig
+          switchLength="120px"
+          onLabel="LT2"
+          offLabel="LT1"
+          fontSize="25px"
+        />
+      </div>
+      <div>
+        <ToggleSwitchBig
+          switchLength="120px"
+          onLabel="Conventional"
+          offLabel="Primitive"
+          fontSize="10px"
         />
       </div>
+      <div style={{ padding: "8px" }}>
+        <ToggleSwitch switchLength="40px" />
+      </div>
+      <div style={{ padding: "8px" }}>
+        <ToggleSwitch thumbClassName="custom-thumb-color" />
+      </div>
+      <div style={{ padding: "8px" }}>
+        <ToggleSwitch switchLength="120px" />
+      </div>
+      <span>FlipperSwitch</span>
+      <div style={{ padding: "4px" }}>
```

Comment:
The labels for this switch have a fairly unintuitive location, which could be improved. Or if we're not really using this switch, it could also be removed.